lms/templates/learner_dashboard/programs.html

```diff
@@ -0,0 +1,26 @@
+<%page expression_filter="h"/>
+<%inherit file="../main.html" />
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.js_utils import (
+    dump_js_escaped_json, js_escaped_string
+)
+%>
+
+<%block name="js_extra">
+<%static:require_module module_name="js/learner_dashboard/program_list_factory" class_name="ProgramListFactory">
+ProgramListFactory({
+    programsData: ${programs | n, dump_js_escaped_json},
+    xseriesUrl: '${xseries_url | n, js_escaped_string}'
+});
+</%static:require_module>
+</%block>
+
+<%block name="pagetitle">${_("Programs")}</%block>
+
+<div class="program-list-wrapper">
+    <div class="program-cards-container"></div>
+    <div class="sidebar"></div>
+    <div class="clearflow"></div>
```

Comment:
Is this DOM element just used for clearfixing? If so it should be removed and the Sass should be updated to achieve the same effect.